src/Api/SecretManagerFeatures/Models/Request/AccessTokenCreateRequestModel.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Bit.Core.Entities;
+using Bit.Core.Utilities;
+
+namespace Bit.Api.SecretManagerFeatures.Models.Request;
+
+public class AccessTokenCreateRequestModel
+{
+    [Required]
+    [EncryptedString]
+    [EncryptedStringLength(200)]
+    public string Name { get; set; }
+    [Required]
+    [EncryptedString]
+    [EncryptedStringLength(4000)]
+    public string EncryptedPayload { get; set; }
+    [Required]
+    [EncryptedString]
+    public string Key { get; set; }
+    [Required]
+    public DateTime ExpireAt { get; set; }
+
+    public ApiKey ToApiKey(Guid serviceAccountId)
+    {
+        return new ApiKey()
+        {
+            ServiceAccountId = serviceAccountId,
+            Name = this.Name,
+            Key = this.Key,
```

Comment:
We tend to not prefix things using `this` in c#.

Reply:
Resolved